utils/index.js

```diff
@@ -132,11 +132,22 @@ const doRedirect = (req, res) => {
 }
 
 // Render a passed-in template and pass in session data under the "data" key
-const renderWithData = template => {
+const renderWithData = (template, { errorsKey } = {}) => {
   return (req, res) => {
-    res.render(template, {
+    let errors = undefined
+
+    // if there are errors in the session under the specified key, add them to the template
```

Comment:
This is to allow us to render errors from another page? (in this case - render DoB errors on the SIN page after the redirect?)

Reply:
Yep, that's what it's for. Only being used by /login/code, and /login/sin